assets/js/components/setup/setup-wrapper.js

```diff
@@ -94,75 +81,66 @@ class SetupWrapper extends Component {
 		}, 500, 'later' );
 	}
 
-	render() {
-		const { currentModule } = this.state;
-		const setupModule = SetupWrapper.loadSetupModule( currentModule );
-		const settingsPageURL = getSiteKitAdminURL(
-			'googlesitekit-settings',
-			{}
-		);
+	const setupModule = loadSetupModule();
```

Comment:
The `loadSetupModule` function isn't needed anymore. Please, render the `SetupComponent` component directly in the returning markup.